packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -753,12 +764,40 @@ export const RemoteServerFactory = {
                     // purpose so we don't want to overwrite the header for those calls.
                     if (incomingRequest.path?.match(options.applySLASPrivateClientToEndpoints)) {
                         proxyRequest.setHeader('Authorization', `Basic ${encodedSlasCredentials}`)
```

Comment:
Can both if block runs at the same time for same path? 

if options.applySLASPrivateClientToEndpoints = '/oauth2\trusted-agent/token' (accidentally), it means, the incoingRe will have both headers. Is that excetable?